source/includes/azure/_security_rules.md

```diff
@@ -65,3 +65,26 @@ Attributes | &nbsp;
 `sourceAddressPrefixes`<br/> *List* | List of IP address ranges or/and IP adresses. If the list is empty then all values are included.
 `destinationPortRanges`<br/> *List* | This specifies on which ports traffic will be allowed or denied by this rule. If the list is empty then all values are included.
 `destinationAddressPrefixes`<br/> *List* | List of IP address ranges or/and IP adresses. If the list is empty then all values are included.
+
+#### Get a security rule
```

Comment:
Did you add this or is this from someone else?

Reply:
yeah this is from PR #100, merging this branch after that one ideally, that one has an update that github isnt showing for some reason